openbb_platform/extensions/platform_api/openbb_platform_api/main.py

```diff
@@ -19,7 +22,20 @@
 CURRENT_USER_SETTINGS = os.path.join(HOME, ".openbb_platform", "user_settings.json")
 USER_SETTINGS_COPY = os.path.join(HOME, ".openbb_platform", "user_settings_backup.json")
 
-FIRST_RUN = True
+# Load widget_exclude_filter.json if it exists
+# Widget filtering is optional and can be used to exclude widgets from the widgets.json file
+# You can generate this filter on OpenBB Hub: https://my.openbb.co/app/platform/widgets
```

Comment:
Can I test this? I currently get a 500 error on both `.co` and `.dev`.

Reply:
Currently available here https://github.com/OpenBB-finance/openbb-hub/pull/398